services/p48-blueapi/README.md

```
@@ -1,27 +1,67 @@
+    - **Beamline Cluster Name** or **Cluster Name**: 
+      `pollux`
     - **Namespace Name:**  
       `p48-beamline`
     - **Ingress Name:**  
       `p48-blueapi.diamond.ac.uk`
 
-Please add the instrument session in the worker.env.metadata.instrument_session
+3. Configure numtracker using the numtracker CLI client:
+  Load the numtracker module on any DLS Red Hat workstation:
 
+  ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)